omnigibson/simulator.py

```diff
@@ -421,6 +421,8 @@ def _non_physics_step(self):
                 # Note that we don't explicitly do for obj in self._objects_to_initialize because additional objects
                 # may be added mid-iteration!!
                 # For this same reason, after we finish the loop, we keep any objects that are yet to be initialized
+                # First call zero-physics step update, so that handles are properly propagated
+                og.sim.pi.update_simulation(elapsedStep=0, currentTime=og.sim.current_time)
```

Comment:
hmm how comes we didn't catch this before?

Reply:
I think it's because we never tried to initialize the edge case of:

not kinematic_only
not fixed_base
not has_articulated_joints
yes has_fixed_joints

AFTER the sim starts playing. Note that if the sim is stopped and we import an object like the above and then play, we never run into issues. I only ran into this issue now because I was testing a manually-defined particleapplier which was imported after sim was playing.

Reply:
test with GPU_DYNAMICS with articulated object, e.g.: robot or cabinet

Reply:
result: doesn't work )):